src/main/java/com/savvato/tribeapp/dto/UserDTO.java

```diff
@@ -34,5 +35,5 @@ public class UserDTO {
   @Schema(example = "2007-12-13T10:15:30")
   public String lastUpdated;
 
-  public Set<UserRole> roles;
+  public List<UserRoleDTO> roles;
```

Comment:
I think a Set would still work here, and it would better communicate the intention that each UserRoleDTO is unique.

Reply:
Was not a small change!!